omero/sysadmins/grid.txt

```diff
@@ -24,6 +24,9 @@ Requirements
 ^^^^^^^^^^^^
 
 The normal OMERO installation actually makes use of OMERO.grid internally.
+Note that multi-node OMERO.grid configurations are only tested with Unix-like
+systems, multi-node configurations on Windows are untested and deprecated,
+and any remaining support will be dropped in OMERO 5.3.
 If you have followed the instructions under :doc:`unix/server-installation`
 (Unix) or :doc:`windows/server-installation` (Windows), then you
```

Comment:
I originally removed that, then put it back in after I realised this page is a bit ambiguous, saying OMERO.grid can be used on the same host as well as across multiple hosts (and there's still an `OMERO.grid on Windows` section on this page). @joshmoore ?